### PR TITLE
Adds php 7.3

### PR DESCRIPTION
**Pull request by kimpepper**
* NEW - Move to https://github.com/skpr/php-alpine
* NEW - Adds PHP 7.3
* LEGACY - Remove the legacy php7.2 image (not used at PNX, checked via Github) **shrinks build times**
* LEGACY - Lock in bundler version for ruby2.1 compatibility.

### CODE REVIEW

File: php/apache/Dockerfile [hunk 1]
```diff
@@ -1,52 +1,56 @@
-ARG ALPINE_VERSION=3.7
+ARG ALPINE_VERSION=3.9
 
 FROM alpine:${ALPINE_VERSION}
+ARG ALPINE_VERSION=3.8
```
Comment (kimpepper) [inline php/apache/Dockerfile]: This default value doesn't match the above.

File: php/apache/Dockerfile [hunk 2]
```diff
@@ -1,52 +1,56 @@
-ARG ALPINE_VERSION=3.7
+ARG ALPINE_VERSION=3.8
 
 FROM alpine:${ALPINE_VERSION}
+ARG ALPINE_VERSION=3.8
```
Comment (kimpepper) [inline php/apache/Dockerfile]: We default to alpine 3.8 here but 3.9 in the dev dockerfile. I realise they are args but would be good to be the same.

File: php/apache/Makefile [hunk 3]
```diff
@@ -44,7 +44,7 @@ define push
 endef
 
 # Build all PHP versions
-build: 7.1 7.2 7.3
```
Comment (kimpepper) [inline php/apache/Makefile]: We still want to build all, right?

File: php/apache/Dockerfile (hunk 2 above)
Comment (nickschuch) [reply php/apache/Dockerfile]: agreed

File: php/apache/Makefile (hunk 3 above)
Comment (nickschuch) [reply php/apache/Makefile]: on it!

**Comment by kimpepper** [comment]
LGTM 👍

**Pull request merged**